source/docs/articles/sites/domains.md

```diff
@@ -64,7 +61,7 @@ Using the provided destinations in the Site Dashboard, create the recommended DN
 To serve your site from the bare domain, you must:
 
 1. Select a DNS provider that supports CNAME flattening, such as [CloudFlare (recommended)](https://support.cloudflare.com/hc/en-us/articles/200169056-CNAME-Flattening-RFC-compliant-support-for-CNAME-at-the-root), [ClouDNS](https://www.cloudns.net/features/), or [NameCheap](https://www.namecheap.com/domains/freedns.aspx).
-2. Do not add the recommended DNS entries from the Dashboard. Instead, create CNAME records for **both** the bare domain (@) and the www subdomain, pointing to `live-yoursite.pantheon.io`.
+2. Do not add the recommended DNS entries from the Dashboard. Instead, create CNAME records for both the bare domain (@) and the www subdomain, pointing to `live-yoursite.pantheon.io`.
```

Comment:
Yes, but for both CNAME records

Brian MacKinney
Developer Training Manager | Pantheon
p 415-800-1957
brian@pantheon.io
www.linkedin.com/in/bmackinney

On Fri, Feb 12, 2016 at 2:52 PM, Rachel <notifications@github.com> wrote:

> In source/docs/articles/sites/domains.md
> <https://github.com/pantheon-systems/documentation/pull/1340#discussion_r52809453>
> :
>
> > @@ -64,7 +61,7 @@ Using the provided destinations in the Site Dashboard, create the recommended DN
> >  To serve your site from the bare domain, you must:
> >
> >  1. Select a DNS provider that supports CNAME flattening, such as [CloudFlare (recommended)](https://support.cloudflare.com/hc/en-us/articles/200169056-CNAME-Flattening-RFC-compliant-support-for-CNAME-at-the-root), [ClouDNS](https://www.cloudns.net/features/), or [NameCheap](https://www.namecheap.com/domains/freedns.aspx).
> > -2. Do not add the recommended DNS entries from the Dashboard. Instead, create CNAME records for **both** the bare domain (@) and the www subdomain, pointing to `live-yoursite.pantheon.io`.
> > +2. Do not add the recommended DNS entries from the Dashboard. Instead, create CNAME records for both the bare domain (@) and the www subdomain, pointing to `live-yoursite.pantheon.io`.
>
> @bmackinney <https://github.com/bmackinney> do you mean a screenshot from
> a DNS provider? I think we try to avoid images from other services since
> they're a pain to keep up to date, but I can add this one from cloudflare
> if you think its needed
> [image: screen shot 2016-02-12 at 2 47 58 pm]
> <https://cloud.githubusercontent.com/assets/10119525/13022734/30b56a04-d198-11e5-8981-0b80645616bb.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pantheon-systems/documentation/pull/1340/files#r52809453>
> .
>